### DISCUSSION TITLE
Reproduce a doc's example

### DISCUSSION THREAD

**Discussion by Ale-Code404** [bcakmakoglu/vue-flow#1114] [Q&A] [answered]
I would to "clone" any example in the documentation, how could i do that?

**Comment by bcakmakoglu** [comment, marked as answer]
Well, sadly the docs examples do not provide a quick-link to a code sandbox or anything (my bad, I've been wanting to implement this for a looong time now but I just never have the time to do so).

[Here's a basic template code sandbox.](https://codesandbox.io/p/sandbox/bold-snow-5x9zjs?file=%2Findex.html%3A1%2C1)
You can use this to copy + paste files from the docs examples into the sandbox.

**Reply by Ale-Code404** [reply]
Hey, thanks for the example, I would like to help you, maybe you could implement the examples documentation to quickly test the Vue Flow functions using what you have in your CodeSanbox account as a base.
